lib/money/money.rb

```diff
@@ -194,6 +194,8 @@ def to_s(style = nil)
     case style
     when :legacy_dollars, nil
       sprintf("%.2f", value)
+    when :with_currency
+      "#{value.to_s('F')} #{currency.to_s}"
```

Comment:
I think the default should have currency before value `USD 10.00`. Did you put it after to `to_f` and `to_i` still work correctly?

Reply:
Haha, that's actually a nice unexpected side effect.

I've looked around and there's no standardized notation for monetary values like ISO8601 for timestamps with timezone. Lacking that, I'm used to writing stuff down in [SI notation](https://en.wikipedia.org/wiki/International_System_of_Units#General_rules), e.g. 1 MHz. It also neatly lines up with the argument order when calling `Money.new`.

Or you could also say this notation betrays my European origins where this is customary ;)

Reply:
Indeed, there's no actual pattern for that. Should we have different outputs depending on which currency?

Reply:
the reason why I'm asking that is because I guess north americanas sorta expect that currency would come first.  Maybe we have a `format` method that I'm not aware of that users can pick the format they want...